weldx/constants.py

```diff
@@ -63,6 +64,9 @@
 """
 
 
+# set default unit registry for pint-xarray
+pint.set_application_registry(WELDX_UNIT_REGISTRY)
```

Comment:
I got an error saying the unit registries are not the same when previously using 
```python
pint_xarray.unit_registry = WELDX_UNIT_REGISTRY
```

I will have to look more into the documentation here but this approach now seems to work as intended @vhirtham